pkg/daemon/agent/agent.go

```diff
@@ -62,7 +63,27 @@ func (a *Agent) Run() error {
 		volumeManager,
 	)
 
-	flexvolumeServer.Start()
+	err = rpc.Register(flexvolumeController)
```

Comment:
this got moved out of the flexvolumeSerer.Start function.  I'm assuming that's because it can only be called once?

Reply:
right, it should only be called once